.github/workflows/on-demand-vrfv2plus-performance-test.yml

```diff
@@ -80,6 +80,8 @@ jobs:
       REF_NAME: ${{ github.head_ref || github.ref_name }}
       CHAINLINK_IMAGE: ${{ inputs.chainlinkImage }}
       CHAINLINK_VERSION: ${{ inputs.chainlinkVersion }}
+      SLACK_API_KEY: ${{ secrets.QA_SLACK_API_KEY }}
+      SLACK_CHANNEL: C0629GXEB0D
```

Comment:
Is this ok to not be a secret?

Reply:
our guess is that its not a secret since it's just an identifier

Reply:
FWIW I was able to use channel id here:
https://github.com/smartcontractkit/chainlink/blob/c1fc9d293ecc5822ec06aa15797d32d8cc10ad83/.github/workflows/ci-core.yml#L107

Reply:
end up adding new secret - `QA_VRF_SLACK_CHANNEL `